lib/deploygate/api/v1/base.rb

```diff
@@ -45,7 +45,11 @@ def post(path, params, &process_block)
         private
 
         def client
-          HTTPClient.new(:agent_name => "dg/#{DeployGate::VERSION}")
+          timeout = 60 * 60 # 1 hour
```

Comment:
Doesn't make any sense. We should cover *realistic* use-cases.

At most 5 minutes is enough to avoid holding unreliable connections. (IMHO, 5 mins sound for me too much though)

Reply:
@jmatsu 
i changed timeout.
please review!